stacker/providers/aws/interactive.py

```diff
@@ -80,7 +80,7 @@ def output_summary(fqn, action, changeset, replacements_only=False):
     changes = []
     for change in changeset:
         resource = change['ResourceChange']
-        replacement = resource['Replacement'] == 'True'
+        replacement = resource.get('Replacement', '') == 'True'
```

Comment:
Right. I got into the habit of making it explicit for clarity, and I thought it would maybe be better to return a string for uniformity, but it's only a matter of preference, and keeping it like the rest of the codebase is best.